src/Workspaces/VisualBasic/Portable/Classification/SyntaxClassification/EmbeddedLanguageTokenClassifier.vb

```diff
@@ -0,0 +1,18 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports System.Collections.Immutable
+Imports Microsoft.CodeAnalysis.Classification.Classifiers
+Imports Microsoft.CodeAnalysis.VisualBasic.EmbeddedLanguages.LanguageServices
+
+Namespace Microsoft.CodeAnalysis.VisualBasic.Classification.Classifiers
+    Friend Class EmbeddedLanguageTokenClassifier
```

Comment:
>EmbeddedLanguageTokenClassifier [](start = 17, length = 31)

nit: should this be prefixed with `VisualBasic`? (and corresponding question for the C# one)